docs/source/tutorials/plots_intro.ipynb

```diff
@@ -530,6 +530,7 @@
    "id": "5211f6de-d447-46a8-8652-3c9fd77d7915",
```

Comment:
<blockquote>The functions used by <code>plot_dist</code> (and by other <code>plot_...</code> functions) aim to be somewhat general, so for example, <code>y</code> is a valid key for encoding information</blockquote><p>Run-on sentence. Split as: ...aim to be somewhat general. For example, <code>y</code> .... </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz-plots/pull/261/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>